src/power-flow-card.ts

```diff
@@ -22,6 +22,7 @@ import { coerceNumber, coerceStringArray, round } from "./utils.js";
 
 const CIRCLE_CIRCUMFERENCE = 238.76104;
 const KW_DECIMALS = 1;
+const W_DECIMALS = 0;
```

Comment:
I don't know... I would prefer to see _XXXW_ rather than _XXX.XW_.
I think, a Watt is already a good enough resolution, plus if you don't have sub-Watt resolution with your sensors, you'd end up with _XXX.0W_ (with a completely useless 0 at the end).
That's also the "why" for the nested if else below